src/sidebar/store/modules/links.js

```diff
@@ -26,20 +16,52 @@ const update = {
 
 const actions = actionTypes(update);
 
-/** Return updated links based on the given current state and action object. */
+/**
+ * Update links
+ *
+ * @param {object} newLinks - Link map returned by the `/api/links` endpoint
+ */
 function updateLinks(newLinks) {
   return {
     type: actions.UPDATE_LINKS,
     newLinks,
   };
 }
 
+/**
+ * Render a service link (URL) using the given `params`
+ *
+ * Returns an empty string if links have not been fetched yet.
+ *
```

Comment:
We don't have a clear convention yet for typing the `state` in store modules. I think we should sort that out first so that the change can be applied consistently going forwards. I would probably suggest something like defining a `State` type near the top of the module.

Reply:
👍🏼